PersonBook/PersonBook/PersonBook/Service/PersonService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using PersonBook.Entity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PersonBook.Service
+{
+    class PersonService : IPersonService<Person>
+    {
+        private List<Person> _personList;
+        private readonly Random _random = new Random();
```

Comment:
You can create a method which can return new id and that method can calculate the id by adding 1 to the biggest id in the list.

Reply:
Done